using-timescaledb/visualizing-data.md

```diff
@@ -0,0 +1,58 @@
+# Visualizing data
+
+The time-series data stored in TimescaleDB is a natural fit for visualization.
+You can use SQL to manipulate data over time and funnel the results into the visualization
+tools or frameworks that you prefer.
+
+TimescaleDB is compatible with visualization tools that work with PostgreSQL. This means that
+regardless of whether you are creating custom visualizations embedded in your applications or
+using off-the-shelf visualization tools to expose data across your business organization, you
+can choose from a wide selection of tools.
+
+## Grafana [](grafana)
+
+Grafana is an open source visualization tool popular in the DevOps monitoring space,
+although it can also be used across the organization to visualize time series metrics.
+Getting started with Grafana is simple. Download and install [Grafana][grafana-install].
+Then, add a new PostgreSQL data source that points to your TimescaleDB instance.
+Queries run through Grafana will continue to benefit from the performance improvements
+built into TimescaleDB.
+
+Once you have connected a data source, you can start building dashboards. Grafana
+allows you to write SQL queries to define dynamic dashboards.
+
+<!-- -->
+>:WARNING: Grafana expects data received to come in timestamp order. When querying
+Grafana using SQL, you must include the `ORDER BY timestamp DESC` statement so that
```

Comment:
The expected ordering is not DESC but ascending so the query bit should be `ORDER BY timestamp`

Reply:
Done!